src/internal/base-component/metrics/log-clients.ts

```diff
@@ -81,6 +89,10 @@ export class PanoramaClient {
     if (typeof metric.eventValue === 'object') {
       metric.eventValue = JSON.stringify(metric.eventValue);
     }
+    if (!validateLength(metric.eventName, 1000)) {
+      console.error(`Event name for metric is too long: ${metric.eventName}`);
```

Comment:
This condition is not covered with u tests

Reply:
Should be fixed now